### PR TITLE
Setup development

### PR DESCRIPTION
**Pull request by lmachens**
this project is using prettier, htmlhint and sass.

image

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -102,3 +102,5 @@ dist
 
 # TernJS port file
 .tern-port
+
+.DS_STORE
```
Comment (phlgr) [inline .gitignore:106]: Add .vscode

**Review by phlgr — CHANGES_REQUESTED** [review]

**Review by nadinho — APPROVED** [review]
lgtm

**Review by Susanne1307 — APPROVED** [review]
👍

**Review by florianGierlichs — APPROVED** [review]
lgtm

**Pull request merged**